changelogs/unreleased/bump-libcurl-to-8.10.1.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/build
+
+* Updated libcurl to version 8.10.1 (gh-10576).
```

Comment:
Short description says that you stick to 8.10.1 + patches, but in commit message itself you say that version is 8.10.1. I propose to stick to a commit tagged as 8.10.1 and fix commit message. I don't see reasons to stick to intermediate commits. Also, it would be useful to describe that updated version contains vulnerability fixes [^1], it is actually the main reason to update curl.

[^1]: https://curl.se/docs/security.html

Reply:
I see, short description was outdated. I removed mention of patches, curl is updated to the commit curl/curl@7eb8c048470e. Added mention about vulnerability fixes. Do I need to specify which vulnerabilities were fixed?

Reply:
It's better to use https://github.com/curl/curl/releases/tag/curl-8_10_1, the commit  is the same, but link highlight that it is tagged commit and moreover shows a changelog for release.